### PR TITLE
background Image updated

### PR DESCRIPTION
**Pull request by simran-kaurbrar**
- background image changed
- recommended tag position adjusted
- placeholder added to searchbar, serach text added to button

### CODE REVIEW

**Review by mgao08 — COMMENTED** [review]
Close and re-open this pull request to go over the CI test, and see if any errors arises.

File: src/pages/Search/Search.module.css [hunk 1]
```diff
@@ -18,14 +18,77 @@
 	margin:5%;
 	display: inline;
  }
+ .background{
+	background-image: url('src/assets/images/background.jpg');
```
Comment (mgao08) [inline src/pages/Search/Search.module.css:22]: Replace the relative path to this external weblink:
https://images.pexels.com/photos/54455/cook-food-kitchen-eat-54455.jpeg

_This change is for the possible file structure change after deployment_

**Review by mgao08 — CHANGES_REQUESTED** [review]
Please see the comment in `src/pages/Search/Search.module.css`

File: src/pages/Search/Search.module.css (hunk 1 above)
Comment (simran-kaurbrar) [reply src/pages/Search/Search.module.css:22]: i adjusted the size of image and also rotated upside down. Can i do that with the external link.

Comment (mgao08) [reply src/pages/Search/Search.module.css:22]: The external image size will be adjusted the same way as the local image, you do need to make some adjustments to rotate the image tho.

**Comment by simran-kaurbrar** [comment]
- I updated the URL of image
- Fixed the footer of card

**Review by mgao08 — APPROVED** [review]

**Pull request merged**